packages/modules/src/modules/banners/common/choiceCard/ChoiceCardInteractive.tsx

```diff
@@ -14,7 +14,11 @@ import type { ReactComponent } from '../../../../types';
 
 const buildStyles = (design: ChoiceCardSettings | undefined, frequencyColumns: number) => {
     const buttonColour = design?.buttonColour ?? 'transparent';
-
+    const buttonTextColour = design?.buttonTextColour ?? '#767676';
```

Comment:
What are these fallback colours? Are they also in the source component?

Reply:
Updated to remove hardcoded colors